pkg/diskmaker/discovery/discovery_test.go

```diff
@@ -13,27 +13,48 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var lsblkOut string
+var blkidOut string
+
+// HelperCommand returns a fake exec.Cmd for unit tests
+func HelperCommand(command string, args ...string) *exec.Cmd {
```

Comment:
does this has to be an exported function?

Reply:
it doesn't. Updated it to be non-exported function.